components/centraldashboard/public/components/activity-view.js

```diff
@@ -2,6 +2,7 @@ import '@polymer/iron-flex-layout/iron-flex-layout-classes.js';
 import '@polymer/iron-ajax/iron-ajax.js';
 import '@polymer/iron-icon/iron-icon.js';
 import '@polymer/iron-icons/iron-icons.js';
+import '@polymer/paper-card/paper-card.js';
```

Comment:
Any reason this is added?

Reply:
It was added for a reason that is no longer required. Removed, good catch!